mantidproject/mantid
# Fix to fit tab error in Engineering Diffraction

**Description of work.**
In a recently merged PR, focused files no longer have a "bankid" property attached to them. This is currently required for them to be loaded into the Fitting tab. This is shortly to be refactored but this fix addresses it in the short term so that the nightly is usable.

**To test:**
1. Create/load a cal, e.g van 236516 ceria 193749
2. Focus a run e.g. 299080
3. make sure that the focused files are saved in both TOF and dSpacing
4. Make sure the files load into the fitting tab with the unit chosen in the combo box
5. Run the system test for FocusBothBanks (on OS witrh different version of gsl to RHEL - e.g. windows)
`systemtest.bat -C debug -R EnginXScriptTest.FocusBothBanks -j 2`

*There is no associated issue.*

*This does not require release notes* because **the bug was introduced within this sprint**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
